addons/mail/models/discuss/discuss_channel.py

```diff
@@ -54,6 +54,7 @@ def _generate_random_token(self):
         ('group', 'Group')],
         string='Channel Type', required=True, default='channel', readonly=True, help="Chat is private and unique between 2 persons. Group is private among invited persons. Channel can be freely joined (depending on its configuration).")
     is_chat = fields.Boolean(string='Is a chat', compute='_compute_is_chat')
+    is_editable = fields.Boolean('Is Editable', compute='_compute_is_editable')
```

Comment:
note/debate: depending on how "enforcing" we want this value to be, we could take it into account inside channel ir rules directly too